metricbeat/module/redis/info/data.go

```diff
@@ -7,11 +7,96 @@ import (
 // Map data to MapStr
 func eventMapping(info map[string]string) common.MapStr {
 
+	// Full mapping from info
 	event := common.MapStr{
-		"version":    info["redis_version"],
-		"mode":       info["redis_mode"],
-		"os":         info["os"],
-		"process_id": info["process_id"],
+		"clients": common.MapStr{
+			"connected_clients":          info["connected_clients"],
```

Comment:
As we can use namespacing, should we remove clients from the field names or keep it in as people are use to it from redis?